2020-07-10 08:34:22.073140: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.0
2020-07-10 08:34:22.078306: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.0
WARNING:tensorflow:From /usr/lib/python3/dist-packages/tensorflow_core/python/compat/v2_compat.py:68: disable_resource_variables (from tensorflow.python.ops.variable_scope) is deprecated and will be removed in a future version.
Instructions for updating:
non-resource variables are not supported in the long term
2020-07-10 08:34:23.368103: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2599990000 Hz
2020-07-10 08:34:23.368329: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x3d8beb0 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-07-10 08:34:23.368352: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-07-10 08:34:23.371180: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcuda.so.1
2020-07-10 08:34:23.470905: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:983] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2020-07-10 08:34:23.471276: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x410a260 initialized for platform CUDA (this does not guarantee that XLA will be used). Devices:
2020-07-10 08:34:23.471287: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): GeForce RTX 2080 with Max-Q Design, Compute Capability 7.5
2020-07-10 08:34:23.471389: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:983] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2020-07-10 08:34:23.471691: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1639] Found device 0 with properties:
name: GeForce RTX 2080 with Max-Q Design major: 7 minor: 5 memoryClockRate(GHz): 1.095
pciBusID: 0000:01:00.0
2020-07-10 08:34:23.471707: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.0
2020-07-10 08:34:23.472559: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcublas.so.10.0
2020-07-10 08:34:23.473274: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcufft.so.10.0
2020-07-10 08:34:23.473443: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcurand.so.10.0
2020-07-10 08:34:23.474399: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcusolver.so.10.0
2020-07-10 08:34:23.475137: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcusparse.so.10.0
2020-07-10 08:34:23.477457: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudnn.so.7
2020-07-10 08:34:23.477528: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:983] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2020-07-10 08:34:23.477838: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:983] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2020-07-10 08:34:23.478090: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1767] Adding visible gpu devices: 0
2020-07-10 08:34:23.808884: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1180] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-07-10 08:34:23.808910: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1186]      0
2020-07-10 08:34:23.808915: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1199] 0:   N
2020-07-10 08:34:23.809059: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:983] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2020-07-10 08:34:23.809393: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:983] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2020-07-10 08:34:23.809681: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1325] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 7077 MB memory) -> physical GPU (device: 0, name: GeForce RTX 2080 with Max-Q Design, pci bus id: 0000:01:00.0, compute capability: 7.5)
TensorFlow:  1.15
Model:       vgg16
Dataset:     imagenet (synthetic)
Mode:        forward only
SingleSess:  False
Batch size:  48 global
48 per device
Num batches: 100
Num epochs:  0.00
Devices:     ['/gpu:0']
NUMA bind:   False
Data format: NCHW
Optimizer:   sgd
Variables:   replicated
AllReduce:   nccl
==========
Generating training model
WARNING:tensorflow:From /home/ubuntu/lambda-tensorflow-benchmark/benchmarks/scripts/tf_cnn_benchmarks/convnet_builder.py:134: conv2d (from tensorflow.python.layers.convolutional) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.keras.layers.Conv2D` instead.
W0710 08:34:23.821710 140421614331712 deprecation.py:323] From /home/ubuntu/lambda-tensorflow-benchmark/benchmarks/scripts/tf_cnn_benchmarks/convnet_builder.py:134: conv2d (from tensorflow.python.layers.convolutional) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.keras.layers.Conv2D` instead.
WARNING:tensorflow:From /usr/lib/python3/dist-packages/tensorflow_core/python/layers/convolutional.py:424: Layer.apply (from tensorflow.python.keras.engine.base_layer) is deprecated and will be removed in a future version.
Instructions for updating:
Please use `layer.__call__` method instead.
W0710 08:34:23.822455 140421614331712 deprecation.py:323] From /usr/lib/python3/dist-packages/tensorflow_core/python/layers/convolutional.py:424: Layer.apply (from tensorflow.python.keras.engine.base_layer) is deprecated and will be removed in a future version.
Instructions for updating:
Please use `layer.__call__` method instead.
WARNING:tensorflow:From /home/ubuntu/lambda-tensorflow-benchmark/benchmarks/scripts/tf_cnn_benchmarks/convnet_builder.py:266: max_pooling2d (from tensorflow.python.layers.pooling) is deprecated and will be removed in a future version.
Instructions for updating:
Use keras.layers.MaxPooling2D instead.
W0710 08:34:23.842392 140421614331712 deprecation.py:323] From /home/ubuntu/lambda-tensorflow-benchmark/benchmarks/scripts/tf_cnn_benchmarks/convnet_builder.py:266: max_pooling2d (from tensorflow.python.layers.pooling) is deprecated and will be removed in a future version.
Instructions for updating:
Use keras.layers.MaxPooling2D instead.
WARNING:tensorflow:From /home/ubuntu/lambda-tensorflow-benchmark/benchmarks/scripts/tf_cnn_benchmarks/convnet_builder.py:408: dropout (from tensorflow.python.layers.core) is deprecated and will be removed in a future version.
Instructions for updating:
Use keras.layers.dropout instead.
W0710 08:34:23.957515 140421614331712 deprecation.py:323] From /home/ubuntu/lambda-tensorflow-benchmark/benchmarks/scripts/tf_cnn_benchmarks/convnet_builder.py:408: dropout (from tensorflow.python.layers.core) is deprecated and will be removed in a future version.
Instructions for updating:
Use keras.layers.dropout instead.
Initializing graph
WARNING:tensorflow:From /home/ubuntu/lambda-tensorflow-benchmark/benchmarks/scripts/tf_cnn_benchmarks/benchmark_cnn.py:2269: Supervisor.__init__ (from tensorflow.python.training.supervisor) is deprecated and will be removed in a future version.
Instructions for updating:
Please switch to tf.train.MonitoredTrainingSession
W0710 08:34:24.007530 140421614331712 deprecation.py:323] From /home/ubuntu/lambda-tensorflow-benchmark/benchmarks/scripts/tf_cnn_benchmarks/benchmark_cnn.py:2269: Supervisor.__init__ (from tensorflow.python.training.supervisor) is deprecated and will be removed in a future version.
Instructions for updating:
Please switch to tf.train.MonitoredTrainingSession
WARNING:tensorflow:From /usr/lib/python3/dist-packages/tensorflow_core/python/ops/array_ops.py:1475: where (from tensorflow.python.ops.array_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.where in 2.0, which has the same broadcast rule as np.where
W0710 08:34:24.029939 140421614331712 deprecation.py:323] From /usr/lib/python3/dist-packages/tensorflow_core/python/ops/array_ops.py:1475: where (from tensorflow.python.ops.array_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.where in 2.0, which has the same broadcast rule as np.where
2020-07-10 08:34:24.043465: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:983] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2020-07-10 08:34:24.043792: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1639] Found device 0 with properties:
name: GeForce RTX 2080 with Max-Q Design major: 7 minor: 5 memoryClockRate(GHz): 1.095
pciBusID: 0000:01:00.0
2020-07-10 08:34:24.043813: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.0
2020-07-10 08:34:24.043834: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcublas.so.10.0
2020-07-10 08:34:24.043844: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcufft.so.10.0
2020-07-10 08:34:24.043853: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcurand.so.10.0
2020-07-10 08:34:24.043863: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcusolver.so.10.0
2020-07-10 08:34:24.043872: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcusparse.so.10.0
2020-07-10 08:34:24.043882: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudnn.so.7
2020-07-10 08:34:24.043925: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:983] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2020-07-10 08:34:24.044241: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:983] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2020-07-10 08:34:24.044504: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1767] Adding visible gpu devices: 0
2020-07-10 08:34:24.044521: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1180] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-07-10 08:34:24.044526: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1186]      0
2020-07-10 08:34:24.044530: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1199] 0:   N
2020-07-10 08:34:24.044585: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:983] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2020-07-10 08:34:24.044879: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:983] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2020-07-10 08:34:24.045151: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1325] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 7077 MB memory) -> physical GPU (device: 0, name: GeForce RTX 2080 with Max-Q Design, pci bus id: 0000:01:00.0, compute capability: 7.5)
2020-07-10 08:34:24.092796: W tensorflow/compiler/jit/mark_for_compilation_pass.cc:1586] (One-time warning): Not using XLA:CPU for cluster because envvar TF_XLA_FLAGS=--tf_xla_cpu_global_jit was not set.  If you want XLA:CPU, either set that envvar, or use experimental_jit_scope to enable XLA:CPU.  To confirm that XLA is active, pass --vmodule=xla_compilation_cache=1 (as a proper command-line flag, not via TF_XLA_FLAGS) or set the envvar XLA_FLAGS=--xla_hlo_profile.
INFO:tensorflow:Running local_init_op.
I0710 08:34:24.111379 140421614331712 session_manager.py:500] Running local_init_op.
INFO:tensorflow:Done running local_init_op.
I0710 08:34:24.138731 140421614331712 session_manager.py:502] Done running local_init_op.
Running warm up
2020-07-10 08:34:24.404654: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcublas.so.10.0
2020-07-10 08:34:24.570442: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudnn.so.7
2020-07-10 08:34:25.727280: W tensorflow/core/common_runtime/bfc_allocator.cc:239] Allocator (GPU_0_bfc) ran out of memory trying to allocate 3.54GiB with freed_by_count=0. The caller indicates that this is not a failure, but may mean that there could be performance gains if more memory were available.
2020-07-10 08:34:25.727323: W tensorflow/core/common_runtime/bfc_allocator.cc:239] Allocator (GPU_0_bfc) ran out of memory trying to allocate 3.54GiB with freed_by_count=0. The caller indicates that this is not a failure, but may mean that there could be performance gains if more memory were available.
2020-07-10 08:34:25.877905: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 2.89G (3108565760 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:26.293247: W tensorflow/core/common_runtime/bfc_allocator.cc:305] Garbage collection: deallocate free memory regions (i.e., allocations) so that we can re-allocate a larger region to avoid OOM due to memory fragmentation. If you see this message frequently, you are running near the threshold of the available device memory and re-allocation may incur great performance overhead. You may try smaller batch sizes to observe the performance impact. Set TF_ENABLE_GPU_GARBAGE_COLLECTION=false if you'd like to disable this feature.
2020-07-10 08:34:26.426921: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 4.89G (5256049408 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:26.427021: W tensorflow/core/common_runtime/bfc_allocator.cc:239] Allocator (GPU_0_bfc) ran out of memory trying to allocate 2.81GiB with freed_by_count=0. The caller indicates that this is not a failure, but may mean that there could be performance gains if more memory were available.
2020-07-10 08:34:26.429945: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 4.89G (5256049408 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:26.429997: W tensorflow/core/common_runtime/bfc_allocator.cc:239] Allocator (GPU_0_bfc) ran out of memory trying to allocate 2.81GiB with freed_by_count=0. The caller indicates that this is not a failure, but may mean that there could be performance gains if more memory were available.
2020-07-10 08:34:26.535146: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 4.89G (5256049408 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:26.535168: W tensorflow/core/common_runtime/bfc_allocator.cc:239] Allocator (GPU_0_bfc) ran out of memory trying to allocate 1.31GiB with freed_by_count=0. The caller indicates that this is not a failure, but may mean that there could be performance gains if more memory were available.
2020-07-10 08:34:26.535708: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 4.89G (5256049408 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:26.535721: W tensorflow/core/common_runtime/bfc_allocator.cc:239] Allocator (GPU_0_bfc) ran out of memory trying to allocate 1.31GiB with freed_by_count=0. The caller indicates that this is not a failure, but may mean that there could be performance gains if more memory were available.
2020-07-10 08:34:26.693584: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 4.89G (5256049408 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:26.693606: W tensorflow/core/common_runtime/bfc_allocator.cc:239] Allocator (GPU_0_bfc) ran out of memory trying to allocate 1.31GiB with freed_by_count=0. The caller indicates that this is not a failure, but may mean that there could be performance gains if more memory were available.
2020-07-10 08:34:26.694352: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 4.89G (5256049408 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:26.694369: W tensorflow/core/common_runtime/bfc_allocator.cc:239] Allocator (GPU_0_bfc) ran out of memory trying to allocate 1.31GiB with freed_by_count=0. The caller indicates that this is not a failure, but may mean that there could be performance gains if more memory were available.
2020-07-10 08:34:26.695250: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 4.89G (5256049408 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:26.695266: W tensorflow/core/common_runtime/bfc_allocator.cc:239] Allocator (GPU_0_bfc) ran out of memory trying to allocate 2.47GiB with freed_by_count=0. The caller indicates that this is not a failure, but may mean that there could be performance gains if more memory were available.
2020-07-10 08:34:26.695830: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 4.89G (5256049408 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:26.695843: W tensorflow/core/common_runtime/bfc_allocator.cc:239] Allocator (GPU_0_bfc) ran out of memory trying to allocate 2.47GiB with freed_by_count=0. The caller indicates that this is not a failure, but may mean that there could be performance gains if more memory were available.
2020-07-10 08:34:27.026679: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 4.89G (5256049408 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:27.026699: W tensorflow/core/common_runtime/bfc_allocator.cc:305] Garbage collection: deallocate free memory regions (i.e., allocations) so that we can re-allocate a larger region to avoid OOM due to memory fragmentation. If you see this message frequently, you are running near the threshold of the available device memory and re-allocation may incur great performance overhead. You may try smaller batch sizes to observe the performance impact. Set TF_ENABLE_GPU_GARBAGE_COLLECTION=false if you'd like to disable this feature.
2020-07-10 08:34:27.061074: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 5.89G (6329791232 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:27.062125: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 5.89G (6329791232 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:27.062746: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 5.89G (6329791232 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:27.063418: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 5.89G (6329791232 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:27.064067: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 5.89G (6329791232 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:27.064631: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 5.89G (6329791232 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:27.065314: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 5.89G (6329791232 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:27.066024: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 5.89G (6329791232 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:27.124150: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 5.89G (6329791232 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:27.124780: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 5.89G (6329791232 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:27.140353: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 5.89G (6329791232 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:27.140991: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 5.89G (6329791232 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:27.156415: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 5.89G (6329791232 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:27.156433: E tensorflow/compiler/xla/service/gpu/cudnn_conv_algorithm_picker.cc:536] Resource exhausted: Out of memory while trying to allocate 633339904 bytes.
2020-07-10 08:34:27.172811: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 5.89G (6329791232 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:27.172827: E tensorflow/compiler/xla/service/gpu/cudnn_conv_algorithm_picker.cc:536] Resource exhausted: Out of memory while trying to allocate 325058560 bytes.
2020-07-10 08:34:27.214359: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 5.89G (6329791232 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:27.214398: E tensorflow/compiler/xla/service/gpu/cudnn_conv_algorithm_picker.cc:536] Resource exhausted: Out of memory while trying to allocate 170917888 bytes.
2020-07-10 08:34:27.230916: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 5.89G (6329791232 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:27.230932: E tensorflow/compiler/xla/service/gpu/cudnn_conv_algorithm_picker.cc:536] Resource exhausted: Out of memory while trying to allocate 170917888 bytes.
2020-07-10 08:34:27.303180: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 5.89G (6329791232 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:27.303854: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 5.89G (6329791232 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:27.304536: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 5.89G (6329791232 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:27.305168: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 5.89G (6329791232 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:27.348875: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 5.89G (6329791232 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:27.349569: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 5.89G (6329791232 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:27.350249: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 5.89G (6329791232 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:27.350900: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 5.89G (6329791232 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:27.428638: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 5.89G (6329791232 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:27.429289: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 5.89G (6329791232 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:27.429924: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 5.89G (6329791232 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:27.430581: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 5.89G (6329791232 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:27.431290: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 5.89G (6329791232 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:27.431982: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 5.89G (6329791232 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:27.432678: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 5.89G (6329791232 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:27.433384: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 5.89G (6329791232 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:27.434080: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 5.89G (6329791232 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:27.434817: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 5.89G (6329791232 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:27.435489: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 5.89G (6329791232 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:27.436143: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 5.89G (6329791232 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:27.503032: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 5.89G (6329791232 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:27.503670: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 5.89G (6329791232 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:27.518709: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 5.89G (6329791232 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:27.519395: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 5.89G (6329791232 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:27.531961: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 5.89G (6329791232 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-07-10 08:34:27.532144: I tensorflow/compiler/jit/xla_compilation_cache.cc:238] Compiled cluster using XLA!  This line is logged at most once for the lifetime of the process.
2020-07-10 08:34:27.532472: W tensorflow/core/framework/op_kernel.cc:1651] OP_REQUIRES failed at xla_ops.cc:502 : Resource exhausted: Out of memory while trying to allocate 427819008 bytes.
INFO:tensorflow:Error reported to Coordinator: <class 'tensorflow.python.framework.errors_impl.ResourceExhaustedError'>, 2 root error(s) found.
(0) Resource exhausted: Out of memory while trying to allocate 427819008 bytes.
[[{{node cluster_0_1/xla_compile}}]]
Hint: If you want to see a list of allocated tensors when OOM happens, add report_tensor_allocations_upon_oom to RunOptions for current allocation info.

[[cluster_0_1/merge_oidx_1/_73]]
Hint: If you want to see a list of allocated tensors when OOM happens, add report_tensor_allocations_upon_oom to RunOptions for current allocation info.

(1) Resource exhausted: Out of memory while trying to allocate 427819008 bytes.
[[{{node cluster_0_1/xla_compile}}]]
Hint: If you want to see a list of allocated tensors when OOM happens, add report_tensor_allocations_upon_oom to RunOptions for current allocation info.

0 successful operations.
0 derived errors ignored.
I0710 08:34:27.533330 140421614331712 coordinator.py:224] Error reported to Coordinator: <class 'tensorflow.python.framework.errors_impl.ResourceExhaustedError'>, 2 root error(s) found.
(0) Resource exhausted: Out of memory while trying to allocate 427819008 bytes.
[[{{node cluster_0_1/xla_compile}}]]
Hint: If you want to see a list of allocated tensors when OOM happens, add report_tensor_allocations_upon_oom to RunOptions for current allocation info.

[[cluster_0_1/merge_oidx_1/_73]]
Hint: If you want to see a list of allocated tensors when OOM happens, add report_tensor_allocations_upon_oom to RunOptions for current allocation info.

(1) Resource exhausted: Out of memory while trying to allocate 427819008 bytes.
[[{{node cluster_0_1/xla_compile}}]]
Hint: If you want to see a list of allocated tensors when OOM happens, add report_tensor_allocations_upon_oom to RunOptions for current allocation info.

0 successful operations.
0 derived errors ignored.
Traceback (most recent call last):
File "/usr/lib/python3/dist-packages/tensorflow_core/python/client/session.py", line 1365, in _do_call
return fn(*args)
File "/usr/lib/python3/dist-packages/tensorflow_core/python/client/session.py", line 1350, in _run_fn
target_list, run_metadata)
File "/usr/lib/python3/dist-packages/tensorflow_core/python/client/session.py", line 1443, in _call_tf_sessionrun
run_metadata)
tensorflow.python.framework.errors_impl.ResourceExhaustedError: 2 root error(s) found.
(0) Resource exhausted: Out of memory while trying to allocate 427819008 bytes.
[[{{node cluster_0_1/xla_compile}}]]
Hint: If you want to see a list of allocated tensors when OOM happens, add report_tensor_allocations_upon_oom to RunOptions for current allocation info.

[[cluster_0_1/merge_oidx_1/_73]]
Hint: If you want to see a list of allocated tensors when OOM happens, add report_tensor_allocations_upon_oom to RunOptions for current allocation info.

(1) Resource exhausted: Out of memory while trying to allocate 427819008 bytes.
[[{{node cluster_0_1/xla_compile}}]]
Hint: If you want to see a list of allocated tensors when OOM happens, add report_tensor_allocations_upon_oom to RunOptions for current allocation info.

0 successful operations.
0 derived errors ignored.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
File "tf_cnn_benchmarks.py", line 73, in <module>
app.run(main)  # Raises error on invalid flags, unlike tf.app.run()
File "/usr/lib/python3/dist-packages/absl/app.py", line 299, in run
_run_main(main, args)
File "/usr/lib/python3/dist-packages/absl/app.py", line 250, in _run_main
sys.exit(main(argv))
File "tf_cnn_benchmarks.py", line 68, in main
bench.run()
File "/home/ubuntu/lambda-tensorflow-benchmark/benchmarks/scripts/tf_cnn_benchmarks/benchmark_cnn.py", line 1884, in run
return self._benchmark_train()
File "/home/ubuntu/lambda-tensorflow-benchmark/benchmarks/scripts/tf_cnn_benchmarks/benchmark_cnn.py", line 2089, in _benchmark_train
return self._benchmark_graph(result_to_benchmark, eval_build_results)
File "/home/ubuntu/lambda-tensorflow-benchmark/benchmarks/scripts/tf_cnn_benchmarks/benchmark_cnn.py", line 2296, in _benchmark_graph
is_chief, summary_writer, profiler)
File "/home/ubuntu/lambda-tensorflow-benchmark/benchmarks/scripts/tf_cnn_benchmarks/benchmark_cnn.py", line 2434, in benchmark_with_session
is_chief))
File "/home/ubuntu/lambda-tensorflow-benchmark/benchmarks/scripts/tf_cnn_benchmarks/benchmark_cnn.py", line 871, in benchmark_one_step
results = sess.run(fetches, options=run_options, run_metadata=run_metadata)
File "/usr/lib/python3/dist-packages/tensorflow_core/python/client/session.py", line 956, in run
run_metadata_ptr)
File "/usr/lib/python3/dist-packages/tensorflow_core/python/client/session.py", line 1180, in _run
feed_dict_tensor, options, run_metadata)
File "/usr/lib/python3/dist-packages/tensorflow_core/python/client/session.py", line 1359, in _do_run
run_metadata)
File "/usr/lib/python3/dist-packages/tensorflow_core/python/client/session.py", line 1384, in _do_call
raise type(e)(node_def, op, message)
tensorflow.python.framework.errors_impl.ResourceExhaustedError: 2 root error(s) found.
(0) Resource exhausted: Out of memory while trying to allocate 427819008 bytes.
[[{{node cluster_0_1/xla_compile}}]]
Hint: If you want to see a list of allocated tensors when OOM happens, add report_tensor_allocations_upon_oom to RunOptions for current allocation info.

[[cluster_0_1/merge_oidx_1/_73]]
Hint: If you want to see a list of allocated tensors when OOM happens, add report_tensor_allocations_upon_oom to RunOptions for current allocation info.

(1) Resource exhausted: Out of memory while trying to allocate 427819008 bytes.
[[{{node cluster_0_1/xla_compile}}]]
Hint: If you want to see a list of allocated tensors when OOM happens, add report_tensor_allocations_upon_oom to RunOptions for current allocation info.

0 successful operations.
0 derived errors ignored.
